readme.md

```diff
@@ -1,5 +1,13 @@
 # Microsoft Azure IoT SDK for .NET
 
+## 2.0.0 clients release notice
```

Comment:
I don't think it is necessary to specify the minor and patch version. Just v2 should be sufficient.